test/nodes/test_reader.py

```diff
@@ -250,6 +254,58 @@ def test_farm_reader_update_params(docs):
         reader.predict(query="Who lives in Berlin?", documents=docs, top_k=3)
 
 
+@pytest.mark.integration
+def test_farm_reader_load_diff_ways(docs):
```

Comment:
Let's create a separate test function for each of the three test cases.

Reply:
Done!